### ISSUE TITLE
Update release

### ISSUE THREAD

**Issue by NN---** [HexWrench/VisualHG2015#6]
VisualHG2015
Update release: https://github.com/HexWrench/VisualHG2015/releases

**Comment by geoffappleford** [comment]
If anyone is interested, here's an updated release. It's built from an unmodified copy of this repository.

https://github.com/geoffappleford/VisualHG2015/releases/tag/3.0.1.2

I got an incompatibility warning when installing in VS2017 Pro but it seems to be working just fine.

**Comment by NN---** [comment]
Works well for me

**Comment by PedroNunoSantos** [comment]
I had some troubles building @geoffappleford version, so i did my own fork. The vsix is available for download there. Got the same warning mentioned when installing, but it works fine. 
Regards.

https://github.com/PedroNunoSantos/VisualHG2015

**Comment by NN---** [comment]
@PedroNunoSantos Thanks !
I suggest making a new repository with active development of this extension to 2017.
There are some small issues I would like to fix and moreover it should be in VS gallery :)

**Comment by PedroNunoSantos** [comment]
Yeah, visualhg should be available for vs 2017 on the gallery, because.. selfish reasons. :)
It would be a real shame let VisualHg die.

**Comment by NN---** [comment]
I think the project needs renaming like VS TortoiseHG integration . Then you know it is a new project :)
